src/header/launchbar/LaunchbarButtonWithNotification.tsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import classNames from 'classnames';
+
+import LaunchbarButton, { LaunchbarButtonProps } from './LaunchbarButton';
+
+import styles from './Launchbar.scss';
+
+type NotificationIndicatorColour = 'red' | 'green';
+
+type LaunchbarButtonWithNotificationProps = LaunchbarButtonProps & {
+  notificationIndicatorColour: NotificationIndicatorColour | null;
```

Comment:
I still feel that calling this property `notification` (red/green/null) would be more concise and cleaner than calling it `notificationIndicatorColour`. First, `notificationIndicatorColour` is longer. Secondly, it implies that there are some attributes of notification other than the colour that get passed to this component, which is not the case. Third, it makes you check on line 47 for the presence of "colour" rather than of the notification. Ugh 🤢

Reply:
I was of this as a `notificationIndicator` more than the notification itself, a bit similar to the notification LED that we have on smartphones. So that's why thought this would make more sense. But as you describe it, seems this itself is the `notification`. Will rename it to that.

Reply:
✅